doc/connection-backoff.md

```diff
@@ -26,9 +26,8 @@ ConnectWithBackoff()
          != SUCCESS)
     SleepUntil(current_deadline)
     current_backoff = Min(current_backoff * MULTIPLIER, MAX_BACKOFF)
-    current_deadline = now() + current_backoff +
-      UniformRandom(-JITTER * current_backoff, JITTER * current_backoff)
-
+    jitter = UniformRandom(-JITTER * current_backoff, JITTER * current_backoff)
+    current_deadline = now() + Min(current_backoff + jitter, MAX_BACKOFF)
```

Comment:
@dklempner @menghanl @zhangkun83 What do you think about this change? What's the behavior in the Go and Java stacks? The idea is to make `MAX_BACKOFF` be a strict upper bound. Previously, the jitter value may result in going over `MAX_BACKOFF`

Reply:
@dgquintas  In case all connections reach this `MAX_BACKOFF`, are they expected to make connection requests at the same time then(without jitter)?

Reply:
That makes sense, thanks @MakMukhi . @dklempner was that the rationale for the current design?

Reply:
Java is conforming to the original design.